fairroot/base/source/FairFileSource.h

```diff
@@ -65,12 +63,7 @@ class FairFileSource : public FairFileSourceBase
     void CreateNewFriendChain(TString inputFile, TString inputLevel);
     TTree* GetInTree() { return fInChain->GetTree(); }
     TChain* GetInChain() { return fInChain; }
-    void CloseInFile()
-    {
-        if (fRootFile) {
-            fRootFile->Close();
-        }
-    }
+    [[deprecated("Use Close() function")]] void CloseInFile() {}
```

Comment:
As you only deprecate it, it should still have the old functionality. So please keep the full implementation.

Reply:
ok, I will change this.